services/attachment.js

```diff
@@ -9,6 +10,26 @@ const attachmentService = {
     return { count, items }
   },
 
+  updateAttachment: async (id, currentUser, fileName) => {
+    const attachment = await Attachment.findById(id).exec()
+
+    if (!attachment) {
+      throw createError(404, DOCUMENT_NOT_FOUND(attachment.modelName))
+    }
+
+    if (currentUser !== attachment.author.toString()) {
+      throw createForbiddenError()
+    }
+
+    if (fileName) {
+      const [fileExtension] = attachment.fileName.split('.').reverse()
+      attachment.fileName = fileName + `.${fileExtension}`
```

Comment:
i believe it should update a file name in the storage as well

Reply:
there's no need to update in storage, cause this field `fileName` is only like alias for attachment that will be shown on UI @Roman-Peretiatko

Reply:
change storage too